<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget:focus{outline: none;} /*去掉选中虚线框(概率出现)*/
QWidget#Widget{
	background-color: rgb(88,82,76);
}
QToolTip {
	color: rgb(223, 223, 223);
	background-color: rgb(64,65,66);
	border:1px solid grey;
	border-radius:2px;/*否则不生效 Why？*/
    opacity:210;
}
/*Slider*/
QSlider::handle {/*设置中间的那个滑动的键*/                           
    background: rgb(90,90,90);
	border-radius:5px;
	border:1px solid rgb(35,35,35);
}
QSlider::handle:vertical{
    height: 9px;
    margin: -4px; /* width */
}
QSlider::handle:horizontal{
    width: 9px;
    margin: -4px; /* height */
}
QSlider::groove:vertical {
    width: 3px;
	border-radius:1px;
	top:5px; bottom: 5px;
}
QSlider::groove:horizontal {
    height: 3px;
	border-radius:1px;
	/*left:4px; right: 4px; 导致move不精确*/
	padding-left:4px;
	padding-right:4px;
}
QSlider::add-page:verical,QSlider::sub-page:horizontal{/*已经划过的从地方*/                          
	background: rgb(200,50,50);
}
QSlider::sub-page:vertical,QSlider::add-page:horizontal{/*还没有滑上去的地方*/
	background:rgb(45,45,45);
}
</string>
  </property>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>238</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgba(150,150,150,100);
}
QPushButton:pressed{
	background-color:rgba(100,100,100,100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/play.ico</normaloff>:/images/play.ico</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="slider_time">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>240</y>
     <width>452</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_volume">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>238</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgba(150,150,150,100);
}
QPushButton:pressed{
	background-color:rgba(100,100,100,100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/volume.ico</normaloff>:/images/volume.ico</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="slider_volume">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>150</y>
     <width>24</width>
     <height>84</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider{
	background: rgba(90,90,90,100);
	border-radius:2px;
}
</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>242</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(15, 15, 15);
	background-color: rgba(126, 126, 126, 80);
	border-radius:4px;
}
QLabel:hover{
	background-color: rgba(126, 126, 126, 175);
}</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="captionBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#captionBar{
	background-color: rgba(74, 69, 64,75);
}</string>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;Consolas&quot;;
    color:rgb(15,15,15);
	border: 0px solid transparent;
    background-color: transparent;
	margin:1px;
}
QPushButton:hover{
	background-color: rgba(180, 50, 50,200);
}

QPushButton:pressed{
	background-color: rgba(235, 20, 20,100);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_min">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;Consolas&quot;;
    color:rgb(15,15,15);
	border: 0px solid transparent;
    background-color: transparent;
	margin:1px;
}
QPushButton:hover{
	background-color: rgba(137, 144, 143,100);
}

QPushButton:pressed{
	background-color: rgba(99, 99, 99,100);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pin">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>3</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgba(150,150,150,100);
}
QPushButton:pressed{
	background-color:rgba(100,100,100,100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/pin_off.ico</normaloff>:/images/pin_off.ico</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_fileName">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>0</y>
      <width>401</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(25,25, 25);
}</string>
    </property>
    <property name="text">
     <string>FileName</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_extend">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgba(150,150,150,100);
}
QPushButton:pressed{
	background-color:rgba(100,100,100,100);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/extend.ico</normaloff>:/images/extend.ico</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	color:rgb(35, 35, 35);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/default_cover.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	color:black;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_song">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>66</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(30, 30, 30);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Song</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_singer">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(30, 30, 30);
}</string>
   </property>
   <property name="text">
    <string>Singer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_album">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>174</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(30, 30, 30);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>『Album』</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_playL">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>100</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:15px;
	/*border: 2px solid rgba(200,200,200,200);*/
}
QPushButton:hover{
	/*background-color:rgba(150,150,150,100);*/
}
QPushButton:pressed{
	background-color:rgba(100,100,100,100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/play.ico</normaloff>:/images/play.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pre">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>125</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgba(150,150,150,125);
}
QPushButton:pressed{
	background-color:rgba(100,100,100,125);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/pre.ico</normaloff>:/images/pre.ico</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>125</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgba(150,150,150,125);
}
QPushButton:pressed{
	background-color:rgba(100,100,100,125);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/next.ico</normaloff>:/images/next.ico</iconset>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_playBack">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(80,80,80,50);
	border-style:flat;
	border-radius:5px;
}
QPushButton:hover{
	background-color:rgba(150,150,150,125);
}
QPushButton:pressed{
	background-color:rgba(100,100,100,125);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/sequence.png</normaloff>:/images/sequence.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_bg</zorder>
  <zorder>btn_play</zorder>
  <zorder>slider_time</zorder>
  <zorder>btn_volume</zorder>
  <zorder>label_time</zorder>
  <zorder>captionBar</zorder>
  <zorder>label_image</zorder>
  <zorder>label_song</zorder>
  <zorder>label_singer</zorder>
  <zorder>label_album</zorder>
  <zorder>btn_playL</zorder>
  <zorder>btn_pre</zorder>
  <zorder>btn_next</zorder>
  <zorder>btn_playBack</zorder>
  <zorder>slider_volume</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
